crmsh/constants.py

```diff
@@ -348,26 +348,26 @@
                             "cluster-name")
 pcmk_version = ""  # set later
 
-container_type = ["docker", "rkt"]
+container_type = ("docker", "podman", "rkt")
```

Comment:
The coming pacemaker 3.0 will drop "rkt". I prefer to drop it in the master branch.

REF: https://projects.clusterlabs.org/w/projects/pacemaker/pacemaker_3.0_changes/pacemaker_3.0_configuration_changes/